### PR TITLE
fix R16 support: addresses #9 and #11

### PR DESCRIPTION
**Pull request by kuenishi**
for https://github.com/basho/basho_bench/pull/87 and https://github.com/basho/riak_cs/pull/677 .

### CODE REVIEW

File: rebar.config.script [hunk 1]
```diff
@@ -0,0 +1,12 @@
+case erlang:system_info(otp_release) =< "R15B01" of
+    true ->
+        CONFIG;
+    false ->
+        HashDefine = [{d,new_hash}],
+        case lists:keysearch(erl_opts, 1, CONFIG) of
+            {value, {erl_opts, Opts}} ->
+                lists:keyreplace(erl_opts,1,CONFIG,{erl_opts,Opts++HashDefine});
+            false ->
+                [{erl_opts, HashDefine}]
```
Comment (kellymclaughlin) [inline rebar.config.script]: This line needs to be:

```
CONFIG ++ [{erl_opts, HashDefine}]
```

**Comment by kellymclaughlin** [comment]
The `rebar` version in this repo currently does not work with the `rebar.config.script`. If you can copy the version from `riak_cs` as part of this PR it works fine.

**Comment by kuenishi** [comment]
Updated as Kelly mentioned. Well, it was `2.1.0-pre` that worked.

**Comment by rzezeski** [comment]
+1

**Pull request merged**